### ISSUE TITLE
ZFS cacti template

### ISSUE THREAD

**Issue by psiriders** [FransUrbo/snmp-modules#2]
Hi,

Have you made any cacti graph templates for  your ZFS module data? This would save us lot of work.

**Comment by FransUrbo** [comment]
No, I'm afraid not :(. I don't use Cacti any more, so I haven't had the time.

But also, I learned from the other SNMP modules, that they are very hard to make in a way that's exportable on my installation and importable on any one elses... There's hardcoded values that needs to be changed etc, so there's very little point.
